package.json

```
@@ -134,7 +134,7 @@
     "socks": "2.8.3"
   },
   "engines": {
-    "node": "^18.18.0 || ^20.9.0 || ^22.0.0"
+    "node": "^18.18.0 || ^20.9.0 || >=21.1.0"
```

Node 21 is end-of-life (https://github.com/nodejs/Release#release-schedule) and unsupported by release-it, so why add it back?

```suggestion
    "node": "^18.18.0 || ^20.9.0 || >=22.0.0"
```